docs/policy.json.5.md

```diff
@@ -271,3 +274,8 @@ selectively allow individual transports and scopes as desired.
     "default": [{"type": "insecureAcceptAnything"}]
 }
 ```
+# SEE ALSO
+  atomic(1)
+
+# HISTORY
```

Comment:
Nit: These two should probably be `##`, i.e. the `<h2>` level.

Reply:
Fixed.